src/MultiDropdown.js

```diff
@@ -3,19 +3,79 @@ import PropTypes from "prop-types";
 import * as countriesCities from "countries-cities";
 import "./MultiDropdown.css";
 import _ from "lodash";
+import CustomDropdown from "./CustomDropdown";
 
 // HINT User CustomDropdown.js here
 
 export default class MultiDropdown extends Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      country: null,
+      cities: null
+    };
+  }
+  selectCountries(country) {
+    this.setState({ country });
+    this.props.onChange({ country, cities: null });
+  }
+  selectGetCities(cities) {
+    this.setState({ cities });
+    this.props.onChange({ country: this.state.country, cities });
+  }
+  onChange(val, countryName) {
+    if (val) {
+      if (countryName == "country") {
+        this.selectCountries(val.value);
+      } else if (countryName == "cities") {
+        this.selectGetCities(val.value);
+      }
+    }
+  }
+  getCountries() {
+    let countries = countriesCities.getCountries().map(selectValue => {
+      return {
+        value: selectValue,
+        label: selectValue
+      };
+    });
+    return countries;
   }
 
-  onChange() {}
+  getCities() {
+    if (this.state.country) {
+      let city = countriesCities
+        .getCities(this.state.country)
+        .map(selectValue => {
+          return {
+            value: selectValue,
+            label: selectValue
+          };
+        });
+      return city;
+    }
+    return [];
+  }
 
   render() {
-    return <div className="MultiDropdown" />;
+    return (
+      <div className="MultiDropdown">
+        <div>
+          <CustomDropdown
+            options={this.getCountries()}
+            onChange={val => this.onChange(val, "country")}
```

Comment:
You're using a raw string here, if you want to follow this approach you should put this in to a constant.